meta-mender-tegra/recipes-mender/tegra-state-scripts/files/redundant-boot-install-script-uboot

```diff
@@ -50,21 +55,31 @@ mount --bind /dev "${mnt}/dev"
 # Mount an overlay for /var/run so the /var/lib/nv_boot_control.conf symlink becomes writable
 mount -t tmpfs tmpfs ${mnt}/run
 
-# Run the update engine in the context of the just-installed rootfs
-# to ensure that the TNSPEC in the config file it uses matches the
-# TNSPEC in the update payload. But first we have to set up the
-# configuration file with the TNSPEC.
-if [ -L "${mnt}/etc/nv_boot_control.conf" -a -x "${mnt}/usr/bin/setup-nv-boot-control" ]; then
-    mkdir -p "${mnt}/run/tegra-nv-bootctrl"
-    if ! chroot "${mnt}" /usr/bin/setup-nv-boot-control; then
-	echo "ERR: could not initialize nv_boot_control.conf in new rootfs" >&2
-    fi
+if [ -f "${mnt}/etc/nv_boot_control.conf" ]; then
+    rm ${mnt}/etc/nv_boot_control.conf
 fi
+
+# # Run the update engine in the context of the just-installed rootfs
+# # to ensure that the TNSPEC in the config file it uses matches the
+# # TNSPEC in the update payload. But first we have to set up the
+# # configuration file with the TNSPEC.
+
+cat ${mnt}${sysconfdir}/nv_boot_control.template >${mnt}${sysconfdir}/nv_boot_control.conf
+
+chmod 0644 "${mnt}${sysconfdir}/nv_boot_control.conf"
```

Comment:
@anishmonachan7 do you understand why the top two lines aren't handled in [setup-nv-boot-control](https://github.com/OE4T/meta-tegra/blob/40701c99c42563a871b8fca0116440f0958a1124/recipes-bsp/tools/setup-nv-boot-control/setup-nv-boot-control.sh.in) for this specific case?